composer.json

```
@@ -40,7 +40,7 @@
     "source": "https://github.com/nextcloud/news/"
   },
   "require": {
-    "php": "^7.2 || 8.0.x",
+    "php": "^7.2 || 8.0.x || 8.1.x",
```

This might be easier 
```suggestion
    "php": "^7.2 || ~8.0",
```